src/util.ts

```diff
@@ -213,7 +213,7 @@ export function symbolDescriptorMatch(query: rt.PartialSymbolDescriptor, sym: rt
 			continue;
 		}
 		if (key === 'package') {
-			if (!sym.package || !packageDescriptorMatch(query.package, sym.package)) {
+			if (!sym.package || !packageDescriptorMatch(query.package as rt.PackageDescriptor, sym.package)) {
```

Comment:
why this cast? Does `packageDescriptorMatch` accept `undefined` as possible value?